src/contents/en-gb/home.yaml

```diff
@@ -46,6 +46,18 @@ contribute:
     link:
       name: Core Technology
       href: /core_technology
+sharing:
+  title:
+    shared: Stand alone
+    tech: technology
+    safe: SAFE Network
+    desc: Use standalone components from the SAFE Network in your own projects.
+  parsec:
+    title: Protocol for Asynchronous, Reliable, Secure and Efficient Consensus
+    desc: PARSEC is an algorithm which allows a section of network nodes to reach consensus on the validity and order of network events by voting on events and using a gossip protocol to disseminate these votes amongst themselves.
```

Comment:
This description goes right to the right hand edge of the screen on a laptop screen at full size.

Reply:
Reproduced this cross browser - it's also easy to resize and end up with the description overhanging the edge of the screen, when every other section of the page has resized appropriately. Or open the page in iPad Pro portrait dimensions - 1024x1366.